sensor/common/sensor/central_sender_impl.go

```diff
@@ -117,6 +127,22 @@ func (s *centralSenderImpl) send(stream central.SensorService_CommunicateClient,
 				s.stopper.Flow().StopWithError(err)
 				return
 			}
+			select {
+			case <-s.stopper.Flow().StopRequested():
+				log.Info("Stop flow requested")
```

Comment:
It looks like debug info logging. Let's switch to debug level or remove.

Reply:
done